### PR TITLE
Resolves #1864,  #1892, #1862, #1752

### LINKED ISSUE #1864: Add image element defaults
First image element in a project should include:
 - Border Color : None
 - Border Thickness : 0px
 - "Opacity" : 1 (update the label)
 - Source : Random from set (depends on ~~#1711~~ #2057)
 - "Corner Radius" - 0% (update the label)

Re-use the last used style for each new element in a project. A huge time saver for users.

Hide border width until a border color is selected

cc @flukeout

### PR DESCRIPTION
**Pull request by k88hudson**
Some changes to editor defaults

### CODE REVIEW

File: www_src/components/el/el.less [hunk 1]
```diff
@@ -73,9 +73,10 @@
   }
 
   // Image
-  .el-image {
+  &.el-image {
     img {
       display: block;
+      max-width: 300px;
```
Comment (k88hudson) [inline www_src/components/el/el.less]: Note: this will need a change in the desktop implementation

File: www_src/components/el/types/image.jsx [hunk 2]
```diff
@@ -2,11 +2,26 @@ var React = require('react/addons');
 var assign = require('react/lib/Object.assign');
 var Spec = require('../../../lib/spec');
 
+var BASE_DEFAULT_URL = 'https://stuff.webmaker.org/webmaker-android/default-images/';
```
Comment (thisandagain) [inline www_src/components/el/types/image.jsx]: Is this evergreen?

Comment (thisandagain) [inline www_src/components/el/types/image.jsx]: Also ... curious if this should just be "webmaker" vs. "webmaker-android" as they are shared assets.

Comment (k88hudson) [inline www_src/components/el/types/image.jsx]: yep, this is a generic CDN we use for webmaker assets

**Comment by thisandagain** [comment]
One minor question / potential nitpick. Otherwise R+

**Pull request merged**